johnsonandjohnson/Bodiless-JS
# Add env variable to switch which site deploys in bodiless

Testing cases

Prereq: npm run setup

1) No env variable set:
- npm run start -- expected Test Site in Edit mode -- PASS
- npm run build & npm run serve -- expected Test Site in Static mode -- PASS

2) env variable set to test-site
- export APP_SITE_NAME=test-site 
- npm run start -- expected Test Site in Edit mode -- PASS
- npm run build & npm run serve -- expected Test Site in Static mode -- PASS

3) env variable set to __cxstarter__
- export APP_SITE_NAME=__cxstarter__
- npm run start -- expected CanvasX Site in Edit mode -- PASS
- npm run build & npm run serve -- expected CanvasX Site in Static mode -- PASS

4) Verified test site is still coming up:
- http://pr-1491-vbuvbay-jvaff4bu65vgm.us-2.platformsh.site/ -- PASS
- http://edit.pr-1491-vbuvbay-jvaff4bu65vgm.us-2.platformsh.site/ -- FAIL
